### PR TITLE
Add Turing elects information

### PR DESCRIPTION
**Pull request by ParadaCarleton**

### CODE REVIEW

**Comment by ParadaCarleton** [comment]
@yebai No changes were made to the previous versions for the information posts. Only addition is the preview at [_posts/2022-06-18-elect-preview.md](https://github.com/TuringLang/turing.ml/pull/32/files#diff-69288e3144091da12a98f637f3f957a3ae9a49aa5498eed28e7252b471921178). Not sure if the site builds correctly, or how to preview that.

File: _partner/turing-elect.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## Why Turing?
+
+Turing.jl is a world-class probabilistic programming language (PPL) that allows users to specify complex models in only a few lines of code. It is the first truly universal PPL capable of expressing any Bayesian model elegantly, and its innovative new design allows for [blazing fast inference](https://arxiv.org/abs/2002.02702) with [state-of-the-art algorithms](http://proceedings.mlr.press/v118/xu20a.htm). All this makes it the tool of choice for new scientists and [Nobel Laureates](https://people.brandeis.edu/~ghall/papers/Yield_Curve_May_10_2021.pdf) alike.
```
Comment (xukai92) [inline _partner/turing-elect.md:22]:
```suggestion
Turing.jl is a world-class probabilistic programming language (PPL) that allows users to specify complex models in only a few lines of code. It is the first truly universal PPL capable of expressing any Bayesian model elegantly, and its innovative new design allows for [blazing fast inference](https://arxiv.org/abs/2002.02702) with [state-of-the-art algorithms](http://proceedings.mlr.press/v118/xu20a.html). All this makes it the tool of choice for new scientists and [Nobel Laureates](https://people.brandeis.edu/~ghall/papers/Yield_Curve_May_10_2021.pdf) alike.
```

File: _posts/2022-06-17-turing-elect.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+title: "Introducing TuringElect"
```
Comment (xukai92) [inline _posts/2022-06-17-turing-elect.md:2]: What's the purpose of this post? Are we calling any help from the community towards the project or so?

Comment (ParadaCarleton) [reply _posts/2022-06-17-turing-elect.md:2]: Nah, just that @yebai asked me to create a post providing information for the community about the project.

**Comment by yebai** [comment]
Closed in favour of https://withdata.io/election/

**Pull request closed without merging**